security/pfSense-pkg-openvpn-client-export/files/usr/local/pkg/openvpn-client-export.inc

```diff
@@ -468,7 +468,7 @@ EOF;
 					$tls_keydir = "key-direction 1{$nl}";
 				}
 				$conf .= $tls_keydir;
-				$conf .= "<{$tls_directive}>{$nl}" . trim(base64_decode($settings['tls'])) . "{$nl}</{$tls_directive}>{$nl}";
+				$conf .= "<{$tls_directive}>{$nl}" . trim(str_replace("\r\n",$nl,base64_decode($settings['tls']))) . "{$nl}</{$tls_directive}>{$nl}";
```

Comment:
You should probably replace `\r\n` by just `\n` before doing that, similar to what happens for advanced options at https://github.com/pfsense/FreeBSD-ports/blob/devel/security/pfSense-pkg-openvpn-client-export/files/usr/local/pkg/openvpn-client-export.inc#L393

That way you can be certain it ends up with consistent line endings. We also have a function called `unixnewlines()` which will convert to unix newlines, but since this package wants a bit more control over the end product newlines that may not be very helpful.